### DISCUSSION TITLE
Accidently removed jwt cert from settings table

### DISCUSSION THREAD

**Discussion by QueenPM** [requarks/wiki#6413] [Help / Questions] [answered]
Im getting the JwtStrategy error because I've accidentally removed the "cert" row from the settings table and now it shows this error when launching the server.

```
2023-05-05T19:24:58.659Z [MASTER] error: Failed to initialize Authentication Strategies: [ ERROR ]
2023-05-05T19:24:58.659Z [MASTER] error: JwtStrategy requires a secret or key
``` 

How am I able to get it to repopulate it/repopulate it myself?

**Comment by NGPixel** [comment, marked as answer]
...and why did you remove that row in the first place?

Install a fresh instance on a different database, then copy the row back into your current one.

**Reply by QueenPM** [reply]
> ...and why did you remove that row in the first place?
> 
> Install a fresh instance on a different database, then copy the row back into your current one.

Thank you for the help. I was able to start my wiki again, however now it throws this warn and I am unable to log in to my admin account.
`[MASTER] warn: secretOrPrivateKey is not valid key material
`

**Reply by NGPixel** [reply]
You're missing other config keys... So it looks like you deleted more than just the cert row.
